<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-RCCV6C2" failures="2" tests="20" name="com.mk.testcases.Herokuapptestcases" time="42.946" errors="0" timestamp="2024-12-04T22:10:35 IST" skipped="0">
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="validateJSPrompt" time="1.749"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="addRemoveElements" time="5.777">
    <failure message="Number of &#039;Delete&#039; buttons remaining is incorrect. expected [3] but found [2]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Number of 'Delete' buttons remaining is incorrect. expected [3] but found [2]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at com.mk.testcases.Herokuapptestcases.addRemoveElements(Herokuapptestcases.java:89)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- addRemoveElements -->
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="checkFirstRowData" time="1.292"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="validateJSConfirm" time="1.100"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="testTableHeaders" time="1.301"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="validateBrokenImages" time="4.018"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="validateJSAlert" time="1.002"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="validateDragAndDrop" time="1.086"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="validateDropdown" time="1.243"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="validateTinyMceEditor" time="2.172"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="verifyTextInFrames" time="2.946"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="testCheckboxRemoveAndAdd" time="8.082"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="validateCheckboxes" time="1.665"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="validateLastNameSorting" time="1.128"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="testNewWindowText" time="2.056"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="validateContextMenuAlert" time="1.387">
    <failure message="Alert not accepted." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Alert not accepted.
at org.testng.Assert.fail(Assert.java:99)
at com.mk.testcases.Herokuapptestcases.validateContextMenuAlert(Herokuapptestcases.java:351)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- validateContextMenuAlert -->
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="validateEmailColumnFormat" time="1.505"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="testShadowDom" time="0.666"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="getextfromShadowDOM" time="0.843"/>
  <system-out/>
  <testcase classname="com.mk.testcases.Herokuapptestcases" name="validateFileUpload" time="1.928"/>
  <system-out/>
</testsuite> <!-- com.mk.testcases.Herokuapptestcases -->
